lib/model.js

```diff
@@ -736,8 +736,11 @@ function handleAtomics(self, where, delta, data, value) {
  */
 
 Model.prototype.$__delta = function() {
-  const dirty = this.$__dirty();
-
+  let dirty = this.$__dirty();
+  if (this.$__.saveOptions && this.$__.saveOptions.pathsToSave) {
```

Comment:
This works, but on further thought I'd prefer if the logic that handles `pathsToSave` was in `$__handleSave()` rather than `$__delta()`. `$__handleSave()` calls `$__delta()` so shouldn't be too hard